### PR TITLE
Update Dask version to 2022.2.1

### PR DESCRIPTION
**Pull request by jsignell**
The last package is building on conda-forge shortly, so I figured I'd put this up.

### CODE REVIEW

**Review by jrbourbeau — APPROVED** [review]
Thanks @jsignell. This should be good to go once `2022.2.1` lands on conda-forge

**Comment by jsignell** [comment]
closing in favor of #206

**Comment by jsignell** [comment]
@jacobtomlinson it looks like the 3.8 base build is failing. I'm not quite sure how to go about debugging.

**Comment by jacobtomlinson** [comment]
The error says `No module named 'pip'` which is a little strange. I've rerun the checks to see if it is a transient error.

**Comment by jsignell** [comment]
Yeah I tested moving the python install up into the top line and it seems to work better locally. Can you see if that change looks sane to you?

File: base/Dockerfile [hunk 1]
```diff
@@ -10,10 +10,8 @@ ENV PATH /opt/conda/bin:$PATH
 ENV PYTHON_VERSION=${python}
 ENV DASK_VERSION=${release}
 
-RUN conda install --yes nomkl cytoolz cmake \
-    && conda install --yes mamba \
+RUN conda install --yes -c conda-forge nomkl cytoolz cmake mamba python=${PYTHON_VERSION} \
```
Comment (jsignell) [inline base/Dockerfile:13]: I moved the python install up, and added `conda-forge` to this line to make sure we don't have different channels.

Comment (jsignell) [reply base/Dockerfile:13]: hmm that doesn't seem to have quite done it.

**Review by jrbourbeau — APPROVED** [review]
Updating the base image version did the trick -- thanks @jsignell!

**Comment by jsignell** [comment]
Thanks @jrbourbeau!!

**Comment by jakirkham** [comment]
There is an autoupdate bot that will make update PRs here

That said, it misses a few details that we haven't managed to sort out yet ( https://github.com/dask/dask-docker/issues/197 ) ( https://github.com/dask/dask-docker/issues/183 ) ( https://github.com/dask/dask-docker/issues/180 )

**Comment by jsignell** [comment]
> There is an autoupdate bot that will make update PRs here
> 
> That said, it misses a few details that we haven't managed to sort out yet ( #197 ) ( #183 ) ( #180 )

yep! I mildly tried to fix the bot, but I think that would require knowing the previous version which I guess we could hard code or pull from main? I did not get very far...

**Comment by jakirkham** [comment]
Yeah I have as well, but I think there was something I wasn't getting right about how the regex got applied

**Comment by jacobtomlinson** [comment]
Thanks for handling this folks!

**Pull request merged**